src/commands/force/user/create.ts

```diff
@@ -36,6 +37,33 @@ interface FailureMsg {
   message: string;
 }
 
+const permsetsStringToArray = (fieldsPermsets: string | string[]): string[] => {
+  if (!fieldsPermsets) return [];
+  return isArray<string>(fieldsPermsets)
+    ? fieldsPermsets
+    : fieldsPermsets.split(',').map((item) => item.replace("'", '').trim());
+};
+
+const lowercaseAllKeys = (input: Record<string, unknown>): Record<string, unknown> => {
+  return Object.keys(input).reduce(function (accum, key) {
+    accum[key.toLowerCase()] = input[key];
+    return accum;
+  }, {});
+};
+
+const standardizePasswordToBoolean = (input: unknown): boolean => {
```

Comment:
@mshanemc input of `undefined` yields true

Reply:
Also going to return true if input is a string === 'False' || 'FALSE' || 'falsE' ....

Reply:
@mshanemc One more `standardizePasswordToBoolean(null)` === true;